src/components/CallToAction/CallToAction.tsx

```diff
@@ -69,6 +70,7 @@ export const CallToAction = (props: CallToActionProps) => {
           'hover:font-medium focus:font-medium',
           className
         )}
+        href={externalLink ? href : undefined}
       >
```

Comment:
nope nope nope, le component `Link` est reservé au liens sur les pages du router Next, ouai c'est piegeux et assez deep mais faut eviter si c'est du external, du coup faut faut que tu fasses ton if en dehors pour distinguer les href "de l'app" et les href "external"

Cf: https://upmostly.com/nextjs/how-to-link-to-an-external-page-in-nextjs

Reply:
@ruizalexandre PI

Reply:
@nigiraudo modifié effectué